src/Posts.jsx

```diff
@@ -210,16 +207,12 @@ query GetFollowingPosts($offset: Int, $limit: Int) {
 
 const loadMorePosts = () => {
   const queryName =
-    state.selectedTab == "following" && accountsFollowing
-      ? "GetFollowingPosts"
-      : "GetPostsQuery";
-  const type =
-    state.selectedTab == "following" && accountsFollowing ? "following" : "all";
+    state.selectedTab == "following" ? "GetFollowingPosts" : "GetPostsQuery";
+  const type = state.selectedTab;
 
   if (
     state.selectedTab == "following" &&
-    accountsSelected &&
-    accountsSelected.length == 0
```

Comment:
This two properties wasn't even declared. Seems like refactoring error or so. I guess it means `accountsFollowing` here. At least everything works fine